Datastructure/TopperStack.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+/**
+ * Created by ugur on 2016-12-26.
+ */
+exports.__esModule = true;
+var Datastructure;
```

Comment:
@uguratar I don't know which naming convention you are using but in below, some of the variable names start with upper letter, some of them start with lower letter, is it the expected naming?

Reply:
You are right, at least they have to be consistent. I will fix this quickly. Thanks @FiratOZ!

Reply:
@FiratOZ :)) These files should not be in the repo from the first place, they are transpiled js files. With a new commit I removed those files. If everything builds properly I'm gong to merge this.

Reply:
Yep it is built. I can merge with your blessing or you can click the button. @FiratOZ

Reply:
wow k